go/vt/logutil/purge.go

```diff
@@ -57,20 +57,23 @@ var levels = []string{"INFO", "ERROR", "WARNING", "FATAL"}
 // relative to now is greater than [cm]timeDelta
 func purgeLogsOnce(now time.Time, dir, program string, ctimeDelta time.Duration, mtimeDelta time.Duration) {
 	current := make(map[string]bool)
+	symlinks := make(map[string]bool)
 	for _, level := range levels {
-		c, err := filepath.EvalSymlinks(path.Join(dir, fmt.Sprintf("%s.%s", program, level)))
+		symPath := path.Join(dir, fmt.Sprintf("%s.%s", program, level))
+		hardPath, err := filepath.EvalSymlinks(symPath)
```

Comment:
This example from the `os` package may be a more reliable way to test for symlink: https://play.golang.org/p/VHdX7KlARqI. In fact, you should probably do check for a `mode.IsRegular()`, which will take care of other weird things like directory, etc.